test/api/core/entrypoints/embedded/index.test.js

```diff
@@ -120,6 +121,10 @@ describe('lib/core/api/core/entrypoints/embedded/index', () => {
     }
   });
 
+  after(() => {
+    mockrequire.stopAll();
```

Comment:
fix a mock-require leakage (failing that, the "mqtt" protocol is still mocked)

That fix prevents the MQTT test below from randomly failing